### PR TITLE
Telaria Bid Adapter

### PR DESCRIPTION
**Pull request by telariaEng**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

We renamed ourselves to Telaria from Tremor Video after the company split. This is the prebid 1.* adapter for Telaria. 

- test parameters for validating bids
```
{
  bidder: 'telaria',
  params: {
      supplyCode: 'ssp-demo-rm6rh',
      adCode: 'ssp-!demo!-lufip'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email: github@telaria.com
- [x] official adapter submission

### CODE REVIEW

**Comment by mkendall07** [comment]
@telariaEng 
You might need to rebase off latest master to fix those circle linting errors.

**Comment by telariaEng** [comment]
@mkendall07 thanks, yeah did that just now.

**Comment by bretg** [comment]
Doc PR at https://github.com/prebid/prebid.github.io/pull/912

File: modules/telariaBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  getUserSyncs: function (syncOptions, serverResponses) {
+    const syncs = [];
+    if (syncOptions.pixelEnabled && serverResponses.length) {
+      try {
+        serverResponses[0].body.ext.telaria.userSync.forEach(url => syncs.push({type: 'image', url: url}));
+      } catch (e) {}
+    }
+    return syncs;
+  },
+  onTimeout: function (timeoutData) {
```
Comment (jaiminpanchal27) [inline modules/telariaBidAdapter.js]: This function is optional, you can remove if not using.

**Comment by telariaEng** [comment]
@jaiminpanchal27 thanks, I removed the unused method and discarded the package-lock changes.

**Pull request merged**